science/esorepo/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 
 name                esorepo
 version             1.1
-revision            1
+revision            2
```

Comment:
I don't see this as necessary. Do people who already have it installed need to rebuild it because something changed on the new repositories?

Reply:
How are people supposed to upgrade to the updated repository URLs otherwise? The esorepo URLs have changed, therefore the revision should be bumped up, no? Or are you suggesting the version number should be bumped?

Reply:
The `version` only needs to be changed if the `version` of the software changed. The `revision` only needs to be changed if this new repository somehow has different software that the user gets installed. For example, If the `license` changes, we do not change the `revision` (or `version` for that matter) because the software is exactly the same.

Reply:
Exactly, the repository has changed. Therefore I bumped the revision number.

Reply:
@mf2k I think the recision change is indeed valid in this case. Note this port is quite different to most others. The port updates the users configuration adding new third party source repos, and this is what is changing. Without the revision bump users would not get their configuration updated.